dask/dataframe/io/parquet.py

```diff
@@ -540,7 +538,7 @@ def read_parquet(path, columns=None, filters=None, categories=None, index=None,
                 categories=categories, index=index)
 
 
-def to_parquet(df, path, engine='auto', compression='default', write_index=None,
+def to_parquet(df, path, engine='auto', compression=None, write_index=None,
```

Comment:
Please revert this change, `compression='default'` means the default compression for the selected backend (see the docstring below). `None` means something else.

Reply:
done